samples/deployment/test-network/installFPC.sh

```diff
@@ -15,16 +15,24 @@ if [[ -z "${FPC_PATH}"  ]]; then
     echo "Error: FPC_PATH not set"
     exit 1
 fi
-
+if [[ -z "${CC_ID}"  ]]; then
+    echo "Error: CC_ID not set"
+    exit 1
+fi
+if [[ -z "${CC_PATH}"  ]]; then
+    echo "Error: CC_PATH not set"
+    exit 1
+fi
 CHANNEL_ID=mychannel
 
 PEERS=("peer0.org1.example.com" "peer0.org2.example.com")
 
 ERCC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 ECC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 
-CC_ID="echo"
-CC_VER="$(cat ${FPC_PATH}/samples/chaincode/${CC_ID}/_build/lib/mrenclave)"
+#CC_ID?=echo
+#CC_PATH?="$(cat ${FPC_PATH}/samples/chaincode/${CC_ID}"
```

Comment:
let remove this instead of commenting it out

Reply:
hahah I guessed you would say that

Reply:
Done!